src/lib/list/list.scss

```diff
@@ -1,7 +1,7 @@
 @import '../core/style/variables';
 @import '../core/style/list-common';
 @import '../core/style/layout-common';
-@import '../divider/divider';
```

Comment:
@crisbeto don't we have a lint check that should have caught this?

Reply:
The lint check only checks the other way around. It reports if you have static CSS within a file starting with an underscore. (e.g. `_divider-offset.scss`)